internal/reflection/reflection_helper.go

```
@@ -735,3 +782,47 @@ const (
+
+// tenantScopedTypes lists the resource types that are scoped to a tenant. Only these types get
+// automatic tenant filter injection on List, automatic tenant field setting on generic Create,
+// and the TENANT column in table output. Types not listed here are platform-scoped.
+var tenantScopedTypes = map[protoreflect.FullName]bool{
+	"osac.public.v1.BareMetalInstance":             true,
+	"osac.public.v1.BareMetalInstanceCatalogItem":  true,
+	"osac.public.v1.BareMetalInstanceTemplate":     true,
+	"osac.public.v1.Cluster":                       true,
+	"osac.public.v1.ClusterCatalogItem":            true,
+	"osac.public.v1.ClusterTemplate":               true,
+	"osac.public.v1.ComputeInstance":                true,
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofmt)